basic_network.go

```diff
@@ -62,6 +62,7 @@ type BasicVideoNetwork struct {
 	msgChans               map[string]chan *Msg
 	transResponseCallbacks map[string]func(transcodeResult map[string]string)
 	relayers               map[relayerID]*BasicRelayer
+	msgCounts              map[Opcode]int64
```

Comment:
Is `msgCounts` only here to facilitate testing? Maybe add a note to this declaration saying as much. Mixing test-specific instrumentation into functional code paths smells a bit funny, but I suppose it's the easiest path forward other than stubbing out each of these switch cases.

Reply:
Oh yeah... I haven't made up my mind yet, but I thought we could add some logging in the node now, and maybe we can use it as debugging info in a beta network when it's deployed.  Of course we still need to add it into the networking layer.